### ISSUE TITLE
Allow to override creating project directory

### ISSUE THREAD

**Issue by muraii** [audreyr/cookiecutter#909]
* Cookiecutter version: 1.5.1
* Template project url: https://github.com/drivendata/cookiecutter-data-science
* Python version: 3.6
* Operating System: macOS 10.10.5

### Description:

Allow user to create project scaffolding into an existing directory.

I would like to set up a project environment using conda environments as well as cookiecutter. Each tool is built to create a project directory and this cannot be overridden in either tool to my understanding. This results in either errors or a clumsy project creation process, e.g.

- run `conda create -n <envname>`
- run cookiecutter with a dummy project name, which creates a directory under <envname>
- copy cookiecutter scaffolding up into parent <envname> directory
- delete cookiecutter dummy project directory

### What I've run:

Having already run `conda create -n playground` and so created the `playground` directory:

`cookiecutter https://github.com/drivendata/cookiecutter-data-science`

`You've cloned /Users/danielblack/.cookiecutters/cookiecutter-data-science before. Is it okay to delete and re-clone it? [yes]:` yes

`project_name [project_name]:` playground

`repo_name [playground]:`

`author_name [Your name (or your organization/company/team)]:` Daniel

`description [A short description of the project.]:` Testing cookiecutter with existing directory

`Select open_source_license:
1 - MIT
2 - BSD
3 - Not open source
Choose from 1, 2, 3 [1]:` 3

`s3_bucket [[OPTIONAL] your-bucket-for-syncing-data (do not include 's3://')]:`

`Select python_interpreter:
1 - python
2 - python3
Choose from 1, 2 [1]:` 2

`Error: "playground" directory already exists`

**Comment by rmedaer** [comment]
Hi @muraii
I made a PR (#907) in order to tackle this (kind of) issue. Basically I added the option "--strip" which allow you to strip cookiecutter directory (e.g. `cookiecutter --strip -o <your-parent-directory> <your-cookiecutter>` ).
With existing option `-f` it allows you to override content of given directory.

Your process would be:

  * run `conda create -n <env name> <conda program>`
  * run `cookiecutter --strip -f -o <conda env path> <cookiecuutter template>`
